### ISSUE TITLE
Plugin conflict in CLion

### ISSUE THREAD

**Issue by artgreen** [mike42/6502-assembly-intellij#1]
I installed the plugin in CLion. When I try to start a new project, CLion is telling me that this plugin conflicts with "C/C++ Language Support by JetBrains".  Is there a way to configure things so this plugin will work in CLion?  Thanks!

**Comment by mike42** [comment]
Hi, seems you've found a bug.

I was able to replicate this problem on a clean install of CLion 212.5080.54 and version 1.2 of this plugin, by simply installing the plugin and restarting. It was not possible to select the 6502 assembly plugin.

![Screenshot from 2021-09-01 08-21-18](https://user-images.githubusercontent.com/2080552/131584152-8e3c1857-46a3-4959-8a13-988d12641984.png)

I tried some tricks with file associations, but CLion is using its built-in assembly language support (left), and not this plugin (right, from a PHPStorm session).

![Screenshot from 2021-09-01 08-14-52](https://user-images.githubusercontent.com/2080552/131584149-fabc7378-3247-4516-bf2b-740a2cdcfa33.png)

I'll do a bit more research on plugin conflicts, but at this stage I'm leaning towards pulling CLion support via a change to `plugin.xml`.
